dask/array/routines.py

```diff
@@ -37,24 +37,58 @@ def result_type(*args):
     return np.result_type(*args)
 
 
-def atleast_3d(x):
-    if x.ndim == 1:
-        return x[None, :, None]
-    elif x.ndim == 2:
-        return x[:, :, None]
-    elif x.ndim > 2:
-        return x
+@wraps(np.atleast_3d)
+def atleast_3d(*arys):
+    new_arys = []
+    for x in arys:
+        x = asarray(x)
```

Comment:
Shouldn't this be `asanyarray`?

Reply:
Have been using `asarray` in other PRs and hadn't heard anything about using `asanyarray` until now. Though would be happy to change if `asanyarray` is preferred.

Reply:
The only real reason to use it is if the input is an array like that isn't a numpy array or a dask array, but is a chunk type we support (e.g. a `np.ma.masked_array`). Since I expect that code path to be very rarely used (if ever) it's not that big of a deal. FWIW numpy uses `asanyarray` here.

Reply:
Ok, will update. Though this may make the type test for the 1 array case a little trickier. That said, sounds like we want to break apart the test anyways. So we can figure it out after.

Reply:
Interesting side note, appears `asanyarray` was not imported into this module.